x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -254,8 +261,8 @@ export const ComplianceDashboard = () => {
                       cspmIntegrationLink
                     )}
                     isIntegrationInstalled={
-                      getSetupStatus.data?.cspm?.status !== 'unprivileged' &&
-                      getSetupStatus.data?.cspm?.status !== 'not-installed'
+                      getSetupStatus.data?.cspm?.status !== 'not-installed' &&
+                      getSetupStatus.data?.cspm?.status !== 'unprivileged'
```

Comment:
Can we store this code in a variable
 const isIntegrationInstalled = !isCspmInstalled && !hasCspmPrivileged
`isIntegrationInstalled={ isIntegrationInstalled}`?

This will allow the code to be more readable and easier to understand.